test/classify.ts

```diff
@@ -72,7 +72,7 @@ describe("The classify endpoint", () => {
     expect(response.body.classifications[2].prediction).to.equal("food"); // pasta
   });
 
-  it("Should contain confidences and labels", async () => {
+  it("Should contain labels", async () => {
     response = await cohere.classify({
```

Comment:
question: do we make actual requests here? I checked the code, and I can't to find any sort of mocking/patching?

Reply:
yep this makes actual requests